modules/tensor_mechanics/src/bcs/Pressure.C

```diff
@@ -14,16 +14,19 @@
 #include "MooseError.h"
 
 registerMooseObject("TensorMechanicsApp", Pressure);
+registerMooseObject("TensorMechanicsApp", ADPressure);
 
+template <bool is_ad>
 InputParameters
-Pressure::validParams()
+PressureTempl<is_ad>::validParams()
 {
-  InputParameters params = IntegratedBC::validParams();
+  InputParameters params = PressureParent<is_ad>::validParams();
   params.addClassDescription("Applies a pressure on a given boundary in a given direction");
   params.addDeprecatedParam<unsigned int>(
       "component", "The component for the pressure", "This parameter is no longer necessary");
   params.addRequiredCoupledVar("displacements",
                                "The string of displacements suitable for the problem statement");
+  params.addParam<Real>("constant", 1.0, "The magnitude to use in computing the pressure");
   params.addParam<Real>("factor", 1.0, "The magnitude to use in computing the pressure");
```

Comment:
Glad to see you're not just blasting the AD use of `constant`, but should one be deprecated since it's duplicative?

Reply:
`constant` makes more sense in my head since it goes along the convention of function, pp, etc, but don't care strongly